__tests__/app.test.js

```diff
@@ -165,4 +165,38 @@ describe('5-get reviews by Id', () => {
       .expect(404)
     })
   })
-        
\ No newline at end of file
+describe('7-post', () => {
+  test('status:201 adds comments to path', () => {
```

Comment:
Really like this test, explicitly expecting the correct author and body but testing the types of the other properties!